devito/finite_differences/differentiable.py

```diff
@@ -143,6 +143,23 @@ def __getattr__(self, name):
             return self._fd[name][0](self)
         raise AttributeError("%r object has no attribute %r" % (self.__class__, name))
 
+    def _eval_subs(self, old, new, **hints):
```

Comment:
I guess the name `_eval_subs` (instead of `_subs`) is what sympy uses underneath ? so they have `subs` (public) and `_subs` + `_eval_subs` (private) ?

Reply:
Forget about it, reverting to previous commit any combination of moving to differentiable is about two times slower than previously